src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Square extends ColorSupplier implements Figure {

```

Comment:
Don't use verbs for class/interface names. In this case, 'Figure' should be an interface, and its name is appropriate, but ensure that the interface name reflects what it represents or does.